### DISCUSSION TITLE
changing payload type from array to anything else.

### DISCUSSION THREAD

**Discussion by hamza-saqib** [spatie/laravel-webhook-server#130] [Q&A]
hi, I wanna send stringified JSON in payload but I cant found a way to do that can anyone help me with that.

**Comment by ssbrandsma** [comment]
Agreed. Perhaps just remove the array type hint on the payload type? the json_encode in CallWebhookJob will do the job 
With the current implementation it doesn't seem to be possible to have plain JSON (without being embedded in an array) as body in the resulting POST request

**Reply by DotNetSimon** [reply]
I was looking into changing the payload type to support raw body, since I'm trying to use the webhook to send XML instead of JSON.
Unfortunately, it seems this is a more complicated issue than it would seem on first glance, since the Signer is also expecting an array.
It could be done without breaking using a 'mixed' or array|string type instead of the current array, but it would make the typing a lot weaker.
@freekmurze do you have any thoughts/suggestions on allowing other body types besides the array being transformed to JSON?
